completions/jupyter.fish

```diff
@@ -25,16 +25,11 @@ end
 function __fish_jupyter_using_command
   set prefix 0
   set include_opts 0
-  getopts $argv | while read -l key option
-    switch $key
-      case _
-        set cmd_to_match $cmd_to_match $option
-      case prefix
-        set prefix 1
-      case include_opts
-        set include_opts 1
-    end
-  end
+  argparse 'i/include_opts' 'p/prefix' -- $argv
```

Comment:
Is it the case that short names `-i` and `-p` exist? If so, this is a good addition.

If they aren't known to exist, we can change this line to `argparse 'include_opts' 'prefix' -- $argv` and I expect that to be fine and accept only the long args.